.github/workflows/remove_review_cron.yml

```
@@ -117,24 +117,27 @@ jobs:
-            SELECT 'DROP DATABASE ' || quote_ident(datname) || ';' FROM to_drop;
-            EOF
-                      done < namespaces_to_delete.txt
-
-                      echo "Planned DROP statements:"
-                      psql -d "$PSQL_DSN" -Atqc "\i $TMP_SQL" | sed 's/^/  /' || true
-
-                      echo "Executing drops..."
-                      psql -d "$PSQL_DSN" -Atqc "\i $TMP_SQL" | psql -d "$PSQL_DSN"
-
-                      echo "Database cleanup finished."
+cat >> "$TMP_SQL" <<EOF
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)